### PR TITLE
update Create Deal View Step 1

### PR DESCRIPTION
**Pull request by Pooh-88**
Create Deal View Step 1

### CODE REVIEW

File: frontend/src/Components/MakeDealForm/DealFormStep1.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    Button,
+    Text
+  } from '@chakra-ui/react';
+
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+function DealFormStep1(props) {
+    const newDate = new Date();
+    // Set default deadline to be a week
```
Comment (kkarnik) [inline frontend/src/Components/MakeDealForm/DealFormStep1.js:19]: Can you make it such that there is no default deadline?

File: frontend/src/firebaseConfig.json [hunk 2]
```diff
@@ -1 +1,9 @@
-../../secrets/firebaseConfig.json
\ No newline at end of file
+{
```
Comment (kkarnik) [inline frontend/src/firebaseConfig.json:1]: Please do not commit firebaseConfig files as these contain sensitive API keys that we do not want exposed to the public.

File: frontend/src/Components/MakeDealForm/index.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+import {useHistory} from "react-router-dom"
+
+import DealFormStep1 from './DealFormStep1';
+import DealFormStep2 from './DealFormStep2';
+import StepsComponent from './StepsComponent';
+
+function MakeDealForm(props) {
+    const newDate = new Date();
+
+    // Set default deadline to be a week
```
Comment (kkarnik) [inline frontend/src/Components/MakeDealForm/index.js:39]: Can you make it such that there is no default deadline?

File: frontend/src/Components/MakeDealForm/DealFormStep1.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    newDate.setDate(newDate.getDate() + 7);
```
Comment (kkarnik) [inline frontend/src/Components/MakeDealForm/DealFormStep1.js:20]: I think this logic is duplicated in MakeDealForm/index.js

**Comment by Ayan-Bandyopadhyay** [comment]
Please attach screenshot(s), that will make the review process much easier. Thanks!

**Comment by Pooh-88** [comment]
I have updated screenshot

File: frontend/src/firebaseConfig.json (hunk 2 above)
Comment (Ayan-Bandyopadhyay) [reply frontend/src/firebaseConfig.json:1]: This file should not be changed. The symlink was removed, let's put it back.

Comment (Ayan-Bandyopadhyay) [reply frontend/src/firebaseConfig.json:1]: What I meant is that the file should only contain the symlink that it contained before: 

../../secrets/firebaseConfig.json

Comment (Ayan-Bandyopadhyay) [reply frontend/src/firebaseConfig.json:1]: Do not commit the API key

File: frontend/src/firebaseConfig.json [hunk 5]
```diff
@@ -1 +1,9 @@
-../../secrets/firebaseConfig.json
```
Comment (Ayan-Bandyopadhyay) [inline frontend/src/firebaseConfig.json:1]: This is the symlink. This file should be reverted back to only contain the symlink

**Comment by Pooh-88** [comment]
image

**Comment by Pooh-88** [comment]
Done.

**Comment by Ayan-Bandyopadhyay** [comment]
Does not compile. Error message: 
`./src/App.js
Module not found: Can't resolve 'chakra-ui-steps'`

You need to update the package.json in src/frontend/

File: frontend/src/App.js [hunk 6]
```diff
@@ -4,32 +4,36 @@ import AccountView from './Components/AccountView'
 import MakeDealForm from './Components/MakeDealForm'
 import Nav from './Components/Nav'
 import {BrowserRouter as Router, Switch, Route, withRouter} from 'react-router-dom'
-import { ChakraProvider } from '@chakra-ui/provider'
-import { theme } from '@chakra-ui/react'
-
+import { ChakraProvider, extendTheme  } from '@chakra-ui/react'
+import { StepsStyleConfig as Steps } from 'chakra-ui-steps';
```
Comment (Ayan-Bandyopadhyay) [inline frontend/src/App.js:8]: Need to delete this line, otherwise it won't compile

**Comment by Ayan-Bandyopadhyay** [comment]
Also you need to use the old make deal form by default, because the new make deal form is incomplete. create a boolean flag that controls which make deal form is shown.
This is known as a "feature flag".

File: frontend/src/firebaseConfig.json [hunk 7]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+../../secret/firebaseConfig.json
```
Comment (Ayan-Bandyopadhyay) [inline frontend/src/firebaseConfig.json:1]: You updated the symlink again, change it back. the directory is "secrets" not secret

**Review by Ayan-Bandyopadhyay — APPROVED** [review]

**Pull request merged**